projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -108,4 +110,8 @@ module.exports = function () {
 	if (useSoy) {
 		cleanSoy();
 	}
+
+	if (process.env.NODE_ENV !== 'development') {
+		await minify();
```

Comment:
I was wondering where in the process would CSS be minified... Since `terser` can only minify JS, where do you see CSS build pipeline going?

Reply:
I focussed specifically on JS because that's what the LPS called for, but was thinking that once @bryceosterhaus's Dart Sass stuff has gone in, we tack on minification to that — depending on how long that work takes, we could look at doing it sooner, but I figured it was the path of least churn to wait.

Reply:
Sounds good!